auth.py

```diff
@@ -59,36 +60,37 @@ def login():
     log = current_app.logger
 
     if current_user.is_authenticated:
-        return redirect(url_for('profile'))
+        return redirect(url_for('root'))
 
-    form = LoginForm(request.form)
+    form = LoginForm()
+    signupform = SignupForm()
```

Comment:
why SignupForm?

Reply:
oh was there for testing, will remove, thanks!